Makefile

```diff
@@ -128,11 +144,14 @@ checks: front_lint
 #------------#
 create_external_volumes:
 	@echo "🥫 Creating external volumes (production only) …"
+	# zfs replications
 	docker volume create --driver=local -o type=none -o o=bind -o device=${MOUNT_POINT}/data html_data
 	docker volume create --driver=local -o type=none -o o=bind -o device=${MOUNT_POINT}/users users
 	docker volume create --driver=local -o type=none -o o=bind -o device=${MOUNT_POINT}/products products
 	docker volume create --driver=local -o type=none -o o=bind -o device=${MOUNT_POINT}/product_images product_images
 	docker volume create --driver=local -o type=none -o o=bind -o device=${MOUNT_POINT}/orgs orgs
+	# local data
+	docker volume create --driver=local -o type=none -o o=bind -o device=${DOCKER_LOCAL_DATA}/po_data po_data
```

Comment:
Should `po_data` really be an external volume with a mount point since the data comes from a ZFS clone / backup already ? I had kept these kind of volumes for data that is mounted to ProductOpener (host --> container); not for mounting from the container to the host (what's the value added in having this data back on the host ?)

Reply:
see https://github.com/openfoodfacts/openfoodfacts-server/pull/6020#discussion_r742082012

Reply:
ok, currently all mounts are done under `/mnt` in the QEMU VM. `/srv` does not exist there which will cause an error when deploying this to pre-prod.

Reply:
Ok, so what is your proposal, I don't have a strong opinion on that. I was just thinking this data would be local to the machine, hence put it in srv.
Do you think it's better create an external volume and mount it ?

@stephanegigandet and @cquest your opinion may be usefull here. I may provide more detail if it's too much obscure to you.

Reply:
If it **needs** to be mounted, I would just mount it to the same ${MOUNT_POINT} as the rest of PO's data ?

Reply:
@ocervell just as I was going to ask again @cquest, I realized that this is not a good option to have a zclone for this in preprod, as in prod, this folder is /srv/off which does not only contains data but also a lot of stuff like the code and so on…

So I think we have to keep it local for now, as I planned initially.